docs/guides/burst_insar_product_guide.md

```diff
@@ -0,0 +1,290 @@
+| Unwrapping type                                           | Phase unwrapping algorithm used                                                                             | mcf                                                                                                         |
+| Phase at Reference Point                                  | Original unwrapped phase value at the reference point (set to 0 in output unwrapped phase raster)           | -4.21967                                                                                                    |
+| Azimuth line of the reference point in SAR space          | Row number (in SAR space) of the reference point                                                            | 2737.0                                                                                                      |
+| Range pixel of the reference point in SAR space           | Column number (in SAR space) of the reference point                                                         | 739.0                                                                                                       |
+| Y coordinate of the reference point in the map projection | Latitude of the reference point in projected coordinates (UTM Zone - meters)                                | 4112453.3223                                                                                                |
+| X coordinate of the reference point in the map projection | Longitude of the reference point in projected coordinates (UTM Zone - meters)                               | 589307.6248                                                                                                 |
+| Latitude of the reference point (WGS84)                   | Latitude of the reference point in WGS84 Geographic Coordinate System (degrees)                             | 37.1542125                                                                                                  |
+| Longitude of the reference point (WGS84)                  | Longitude of the reference point in WGS84 Geographic Coordinate System (degrees)                            | 40.00574707                                                                                                 |
+| Unwrapping threshold                                      | Minimum coherence required to unwrap a given pixel                                                          | none                                                                                                        |
+| Speckle filter                                            | Speckle filter applied                                                                                      | no                                                                                                          |
```

Comment:
Updating to match example output
[S1_136231_IW2_20200604_20200616_VV_INT80_D94E.txt](https://github.com/ASFHyP3/hyp3-docs/files/12693515/S1_136231_IW2_20200604_20200616_VV_INT80_D94E.txt)

```suggestion
| InSAR phase filter                                        | Name of the phase filter used                                                                               | yes                                                                                                         | 
| Phase filter parameter                                    | Dampening factor                                                                                            | 0.5                                                                                                         |
| Range bandpass filter                                     | Range bandpass filter applied                                                                               | no                                                                                                          |
| Azimuth bandpass filter                                   | Azimuth bandpass filter applied                                                                             | no                                                                                                          |
| DEM source                                                | DEM used in processing                                                                                      | GLO-30                                                                                                      |
| DEM resolution                                            | Pixel spacing in meters for DEM used to process this scene                                                  | 30                                                                                                          |
| Unwrapping type                                           | Phase unwrapping algorithm used                                                                             | snaphu_mcf                                                                                                  |
| Speckle filter                                            | Speckle filter applied                                                                                      | yes                                                                                                         |
```